main.tf

```diff
@@ -1,3 +1,8 @@
+provider "helm" {
```

Comment:
The `helm` provider should be declared in the example, but not in the reusable module.  But moving it out might mean that we don't need the `kubeconfig_local_path` variable?

Reply:
You can't declare it in the example, it says:
So at this point, there is nothing coming down from the addons module, should the helm examples just be a separate repository?

Reply:
I think keeping modules like this in the add-ons has value in that it gets us a standardized approach to configuring k8s add-ons, so someone who wants to install CPEM, longhorn, et al., can do it all through the add-ons module.  (Is it useful to be able to do that?  I don't know; this project is currently experimental, so we need to put together a library of add-ons in order to see if we can build meaningful examples out of it.)

Ideally, though, an add-on would bring some value beyond running a helm install...for longhorn, are there any things we could check/validate about the cluster or machines before installing?